dev_guide/managing_images.adoc

```diff
@@ -329,6 +329,12 @@ link:#creating-an-image-stream-by-manually-pushing-an-image[create an image
 stream by manually pushing an image], or just to `docker pull` an image
 directly.
 
+ifdef::openshift-online[]
+{product-title} (Next Gen) Developer Preview gives developers a hands-on preview
+of the OpenShift platform in a hosted environment that includes access to an
+internal registry located at *_registry.preview.openshift.com_*.
+endif::[]
```

Comment:
:+1: Maybe below where we show the `docker login` command, we can do a separate ifdef for online where we hard-code in https://registry.preview.openshift.com as seen in https://blog.openshift.com/getting-started-docker-registry/